lib/autoyast.pm

```diff
@@ -713,9 +713,13 @@ sub upload_profile {
     my $profile = $args{profile};
     my $path = $args{path};
 
-    if (check_var('IPXE', '1')) {
-        $path = get_required_var('SUT_IP') . $path;
-    }
+    # Don't know why the 'if' statement is required
+    # it fails to save the autoyast profile to the job URL on the worker
+    # fg. http://openqa.suse.de/tests/11712594#step/installation/1
+    # so comment out this part
+    # if (check_var('IPXE', '1')) {
+    #     $path = get_required_var('SUT_IP') . $path;
```

Comment:
Reproduced the issue when including IPXE=1, it generates the `$path` likes`${SUT_IP}autoyast_hanaperf/mu.xml`, but no code loading autoinst.xml following it.
I agree to remove this code block.

Reply:
Will it change the path for kernel team? Better to involve kernel team PO for review when PR is ready.

Reply:
Finally I made this change limited to virt tests and hana tests only.